mypy/checkexpr.py

```diff
@@ -1608,13 +1608,17 @@ def analyze_super(self, e: SuperExpr, is_lvalue: bool) -> Type:
                         return AnyType()
                     if not self.chk.in_checked_function():
                         return AnyType()
-                    selftype = self.chk.function_stack[-1].arguments[0].variable.type
-                    return analyze_member_access(name=e.name, typ=fill_typevars(e.info), node=e,
+                    # fill_typevars(e.info) erases type variables
```

Comment:
Oh. wait. it's only confusing because we talk about different variables... sorry. I'll update.